staff/mary-quiroz/bash/README.md

```diff
@@ -0,0 +1,83 @@
+##Git Bash Commands
+
+## git init (Initializes the new git repository).
+
+```sh
+$ git init
+```
+
+## pwd (path to working directory)
+
+```sh
+It shows us the current folder we are in.
```

Comment:
Each command definition and examples should be written outside the back ticks 🤯

Reply:
done!